atom/common/api/api.mojom

```diff
@@ -11,6 +11,8 @@ interface ElectronRenderer {
       mojo_base.mojom.ListValue arguments,
       int32 sender_id);
 
+  UpdateCrashpadPipeName(string pipe_name);
```

Comment:
It's possible that crashpad will start after the renderer processes have started, so it cannot be a command line arg.
@zcbenz can probably answer better.

Reply:
Yeah, it is especially true in our own tests, as the crashReporter in main process is only started after the renderer process has landed related tests.

Reply:
Hmm, IMO we should change our API such that crashpad is always initialized in the main process and always before renderer processes are started (as in Chrome)

Reply:
We need to give users a chance to setup product name, crash server URL, etc. and it can not be done after the crash reporter is started.

CEF works around it by reading crash reporter settings from an INI config file, we can probably do the same by reading from `package.json`. And yeah it should be a followup change.

Reply:
I agree, setting those values in `package.json` would be a good idea. Extra parameters /etc. that could be runtime dependent could be amended after the JS is run, but at least the crash URL should be set up statically.

Reply:
we are doing this in Teams, we've modified the crashReporter to be only initialized once in the main process and all renderer then inherit the same config. I am planning to upstream this change.